ext/TrixiConvexECOSExt.jl

```diff
@@ -61,6 +61,60 @@ function stability_polynomials!(pnoms, consistency_order, num_stage_evals,
     end
 end
 
+# Specialized form of the stability polynomials for fourth-order PERK schemes.
+function stability_polynomials_PERK4!(pnoms, num_stage_evals,
+                                      num_eig_vals,
+                                      normalized_powered_eigvals,
+                                      gamma,
+                                      dt, cS3)
```

Comment:
Is this basically an optimized version of `stability_polynomials!` specifically for the PERK4 family or does it do something that cannot be achieved using the existing function?